src/insert-expand-diff/insert-expand-diff.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+// @jsx h
+
+import { h } from 'dom-chef'
+
+const onClick = (diff: HTMLElement) => {
+    const expandElipses = diff.querySelectorAll('.ellipsis')
+
+    if (!expandElipses) return
+
+    expandElipses.forEach(expander => expander.click())
+}
+
+export default function insertExpandDiff(diff: HTMLElement) {
+    const button = (
+        <button
+            type="button"
+            class="aui-button aui-button-subtle devtools-task-in-progress--button __rbb-btn-expandDiff"
+            title="Expand unexpanded diff sections once"
+            original-title="Expand unexpanded diff sections once"
+            style={{ position: 'relative' }}
```

Comment:
How would you feel if we add some styles here to make the two buttons look more compact ? Right now there's a lot of extra spacing on the new one that makes it look asymmetric

Now

![image](https://user-images.githubusercontent.com/7514993/100521969-abec5f80-317d-11eb-8b00-ffa429e1bdb4.png)

After

![image](https://user-images.githubusercontent.com/7514993/100521962-a55de800-317d-11eb-9cbe-4f28c532576f.png)

Styles I used to take the second screenshot:

```css
    margin: 0;
    padding-left: 5px;
    padding-right: 5px;

```

It looks great in the new UI though!

Reply:
makes sense!